<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="m_sendEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_gravity">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>69</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>gravity</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_roll">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>69</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>roll</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_pitch">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>69</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>pitch</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_yaw">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>69</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>yaw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_connectButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_disconnectButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_sendButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_enableLoggingCb">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Enable logging the data to a file."/>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_logFileLe">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_logFileFileDialog">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>390</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="m_receiveEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_label_debug">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>69</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
